CRM/Member/BAO/Query.php

```diff
@@ -312,6 +312,18 @@ public static function whereClauseSingle(&$values, &$query) {
         }
         $query->_tables['civicrm_membership'] = $query->_whereTables['civicrm_membership'] = 1;
         return;
+
+      case 'member_override':
```

Comment:
I think this is probably better put as
```php
        $query->_where[$grouping][] = CRM_Contact_BAO_Query::buildClause("civicrm_membership.is_override", $op, $value, "Boolean");
        if ($value) {
          $query->_qill[$grouping][] = ts("Membership Has Status Override Set");
        }
        else {
          $query->_qill[$grouping][] = ts("Membership Does Not Have Status Override Set");
        }
        $query->_tables['civicrm_membership'] = $query->_whereTables['civicrm_membership'] = 1;
```
what you think @colemanw @eileenmcnaughton